<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>465</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>465</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Poppins</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: white;
border-radius: 30px;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>461</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
color:  #383838;
font-weight: 800;
font-family: 'Inter 24pt ExtraBold';
font-size: 	14pt;
background: none;
qproperty-alignment: 'AlignCenter';

</string>
   </property>
   <property name="text">
    <string>CREATE AN ACCOUNT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
font-family: 'Poppins';
font-size: 11pt;
color: #383838;
qproperty-alignment: 'AlignLeft';
border: none;
border-radius: 0;
background: none;
}
</string>
   </property>
   <property name="text">
    <string>First Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
font-family: 'Poppins';
font-size: 11pt;
color: #383838;
qproperty-alignment: 'AlignLeft';
border: none;
border-radius: 0;
background: none;
}
</string>
   </property>
   <property name="text">
    <string>Last Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>381</width>
     <height>39</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border:none;
font-family: 'Poppins';
font-size: 9pt;
background: none;
background-color: #3a2f86;
border-radius: 10px;
color: white;
}

QPushButton:hover {
background-color: #231c53;
}



</string>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mn_line">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>181</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
border: 1px solid #383838;
background: #FFFFF;
font-family: 'Poppins';
padding-left: 10px;
font-size: 11pt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fn_line">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>181</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
border: 1px solid #383838;
background: #FFFFF;
font-family: 'Poppins';
padding-left: 10px;
font-size: 11pt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
font-family: 'Poppins';
font-size: 11pt;
color: #383838;
qproperty-alignment: 'AlignLeft';
border: none;
border-radius: 0;
background: none;
}
</string>
   </property>
   <property name="text">
    <string>Middle Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ln_line">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>181</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
border: 1px solid #383838;
background: #FFFFF;
font-family: 'Poppins';
padding-left: 10px;
font-size: 11pt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
font-family: 'Poppins';
font-size: 11pt;
color: #383838;
qproperty-alignment: 'AlignLeft';
border: none;
border-radius: 0;
background: none;
}
</string>
   </property>
   <property name="text">
    <string>Date of Birth</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
font-family: 'Poppins';
font-size: 11pt;
color: #383838;
qproperty-alignment: 'AlignLeft';
border: none;
border-radius: 0;
background: none;
}
</string>
   </property>
   <property name="text">
    <string>Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="address_line">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>381</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
border: 1px solid #383838;
background: #FFFFF;
font-family: 'Poppins';
padding-left: 10px;
font-size: 11pt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backtologin_btn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>430</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>58</red>
         <green>47</green>
         <blue>134</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>58</red>
         <green>47</green>
         <blue>134</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>58</red>
         <green>47</green>
         <blue>134</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>58</red>
         <green>47</green>
         <blue>134</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>58</red>
         <green>47</green>
         <blue>134</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>58</red>
         <green>47</green>
         <blue>134</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>58</red>
         <green>47</green>
         <blue>134</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>58</red>
         <green>47</green>
         <blue>134</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover {
    text-decoration: underline;
}</string>
   </property>
   <property name="text">
    <string>&lt;  Back to Login</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dob">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>181</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
border: 1px solid #383838;
background: #FFFFF;
font-family: 'Poppins';
padding-left: 10px;
font-size: 11pt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
